plugins/newsdownloader.koplugin/main.lua

```diff
@@ -212,28 +212,38 @@ end
 
 function NewsDownloader:processFeedSource(url, limit, unsupported_feeds_urls, download_full_article)
 
-    local response = DownloadBackend:getResponseAsString(url)
+    local ok, response = pcall(function()
```

Comment:
```lua
pcall(DownloadBackend.getResponseAsString, url)

Reply:
replacing 
```
pcall(function()
        return DownloadBackend:getResponseAsString(url)
    end)
```

with `pcall(DownloadBackend.getResponseAsString, url)`

brokes code. 
It displays error for all sources after <1 sec.